pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -249,6 +234,10 @@ func (b *BridgePodInterface) preparePodNetworkInterfaces() error {
 		return err
 	}
 
+	if err := Handler.NeighDelete(b.podInterfaceName, b.vif.MAC.String()); err != nil {
```

Comment:
What does `NeighDelete` mean? I can't make sense out of it with my english skills :)

Reply:
neighborDelete 

just like there is append here:
https://github.com/vishvananda/netlink/blob/cb78b187011e68135f3b71a3566bbc478087bbc1/neigh_linux.go#L96